tests/core/utils.py

```diff
@@ -1,11 +1,167 @@
 # First Party
+# Standard Library
+import glob
+import json
+import logging
+import os
+import shutil
+from os.path import join
+
 from smdebug.core.collection import Collection
 from smdebug.core.collection_manager import CollectionManager
-from smdebug.core.config_constants import DEFAULT_COLLECTIONS_FILE_NAME
+from smdebug.core.config_constants import (
+    DEFAULT_COLLECTIONS_FILE_NAME,
+    DEFAULT_SAGEMAKER_METRICS_PATH,
+)
+from smdebug.core.modes import ModeKeys
+from smdebug.core.reader import FileReader
+from smdebug.core.sagemaker_utils import is_sagemaker_job
+from smdebug.trials import create_trial
 
 
 def write_dummy_collection_file(trial):
     cm = CollectionManager()
     cm.create_collection("default")
     cm.add(Collection(trial))
     cm.export(trial, DEFAULT_COLLECTIONS_FILE_NAME)
+
+
+def delete_local_trials(local_trials):
+    for trial in local_trials:
+        shutil.rmtree(trial)
+
+
+# need this to seek to the right file offset for test output verification
+metrics_file_position = 0
```

Comment:
why is this global? Can't we make it a class?

Reply:
Fixed.